src/routes/v0/tag/global.rs

```diff
@@ -51,18 +51,21 @@ pub async fn hot_tags_handler(Query(query): Query<HotTagsQuery>) -> Result<Json<
     }
 }
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Debug)]
 pub struct TagTaggersQuery {
+    pagination: Pagination,
+    user_id: Option<String>,
     reach: Option<TagStreamReach>,
 }
 
 #[utoipa::path(
     get,
     path = TAG_TAGGERS_ROUTE,
-    tag = "Global tag Taggers",
+    tag = "Tag Taggers",
     params(
         ("label" = String, Path, description = "Tag name"),
-        ("reach" = TagStreamReach, Path, description = "Reach type: Follower | Following | Friends")
+        ("reach" = TagStreamReach, Path, description = "Reach type: Follower | Following | Friends"),
```

Comment:
that param should be optional. It is but in swagger does not reflect. Apart of that we should do a `reach` and `user_id` error control before call to the model to fetch the info.

```rust
// XOR operation maybe?
if query.reach.is_some() ^ query.user_id.is_some() {
    // Bad request error
}

Reply:
It shouldn't be optional. These are both in the Path and not the query. so the check is also not needed.